.github/workflows/build.yml

```diff
@@ -8,15 +8,17 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
```

Comment:
@endrl I doubt maybe but does this line seems fair or not ?

Reply:
Plugins run cross platform and i have no special requirement. The jf plugin template runs just on [ubuntu](https://github.com/jellyfin/jellyfin-meta-plugins/blob/master/.github/workflows/build.yaml).

Reply:
Solved